cleo/color.py

```diff
@@ -141,9 +141,9 @@ def _degrade_hex_color_to_ansi(self, r: int, g: int, b: int) -> int:
         return (round(b / 255) << 2) | (round(g / 255) << 1) | round(r / 255)
 
     def _get_saturation(self, r: int, g: int, b: int) -> int:
-        r = r / 255
-        g = g / 255
-        b = b / 255
+        r = round(r / 255)
```

Comment:
is this right? rounding here changes the result doesn't it?

Reply:
it would slightly in some situations, yes. have reverted.